packages/react-native-gesture-handler/src/v3/hooks/useGesture.ts

```diff
@@ -94,21 +94,17 @@ export function useGesture<THandlerData, TConfig>(
   }, [type, tag]);
 
   useEffect(() => {
+    const preparedConfig = prepareConfig(config);
+    RNGestureHandlerModule.setGestureHandlerConfig(tag, preparedConfig);
+    RNGestureHandlerModule.flushOperations();
+
```

Comment:
I think it would look better if `bindSharedValues` would be called first, but I don't think that this order matters so we can leave that 😄